src/Book.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Date;
+
+/**
+ * Created by mr-mou on 04/07/17.
+ */
+public class Book {
```

Comment:
Although you have identified two objects so far Book and User there is also one more I found. hint use the LibraryApp class for only controlling the UI